client/python/test/test_client.py

```diff
@@ -1240,7 +1240,7 @@ def test_download_private_file(self, gradio_temp_dir):
             src="gradio/zip_files",
         )
         url_path = "https://gradio-tests-not-actually-private-spacev4-sse.hf.space/file=lion.jpg"
-        file = client.endpoints[0]._upload_file(url_path)  # type: ignore
+        file = client.endpoints[0]._upload_file(url_path, 0)  # type: ignore
```

Comment:
This test is not related to this but needed to be fixed since we changed the signature of `_upload_file`. It wasn't caught before because this test is marked flaky